contracts/contracts/svg/partials/background.hbs

```diff
@@ -1,116 +1,41 @@
-  </g>
-{{~/if~}}
-{{~#if bgRadial5}}
-  <g style="filter: hue-rotate({{uint16 hue}}deg);">
-    <rect style="fill:rgba(225, 114, 127, 1)" width="2000" height="3000"></rect>
-    <radialGradient id="grad5">
-      <stop offset="0" style="stop-color:rgba(255, 255, 255, 1)"></stop>
-      <stop offset="1" style="stop-color:rgba(225, 114, 127, 0)"></stop>
-    </radialGradient>
-    <circle style="fill:url(#grad5)" cx="1000" cy="1925" r="1133"></circle>
-    <circle style="fill:url(#grad5)" cx="1000" cy="372" r="1133"></circle>
-  </g>
-{{~/if~}}
-{{~#if bgRadial6}}
-  <g style="filter: hue-rotate({{uint16 hue}}deg);">
-    <rect style="fill:rgba(13, 17, 24, 1)" width="2000" height="3000"></rect>
-    <radialGradient id="grad6">
-      <stop offset="0" style="stop-color:rgba(168, 224, 126, 1)"></stop>
-      <stop offset="1" style="stop-color:rgba(168, 224, 126, 0)"></stop>
-    </radialGradient>
-    <circle style="fill:url(#grad6)" cx="1000" cy="1925" r="1133"></circle>
-    <circle style="fill:url(#grad6)" cx="1000" cy="372" r="1133"></circle>
-  </g>
-{{~/if~}}
-{{~#if bgLinear0}}
-  <g style="filter: hue-rotate({{uint16 hue}}deg);">
-    <linearGradient id="lin0" gradientTransform="rotate(90)">
-      <stop offset="0%" stop-color="rgba(8, 14, 37, 1)"></stop>
-      <stop offset="100%" stop-color="rgba(115, 131, 186, 1)"></stop>
-    </linearGradient>
-    <rect style="fill:url(#lin0)" width="2000" height="3000"></rect>
-  </g>
-{{~/if~}}
-{{~#if bgLinear1}}
-  <g style="filter: hue-rotate({{uint16 hue}}deg);">
-    <linearGradient id="lin1" gradientTransform="rotate(90)">
-      <stop offset="0%" stop-color="rgba(8, 14, 37, 1)"></stop>
-      <stop offset="100%" stop-color="rgba(111, 92, 147, 1)"></stop>
-    </linearGradient>
-    <rect style="fill:url(#lin1)" width="2000" height="3000"></rect>
-  </g>
-{{~/if~}}
-{{~#if bgLinear2}}
-  <g style="filter: hue-rotate({{uint16 hue}}deg);">
-    <linearGradient id="lin2" gradientTransform="rotate(90)">
-      <stop offset="0%" stop-color="rgba(8, 14, 37, 1)"></stop>
-      <stop offset="100%" stop-color="rgba(247, 38, 43, 1)"></stop>
-    </linearGradient>
-    <rect style="fill:url(#lin2)" width="2000" height="3000"></rect>
-  </g>
-{{~/if~}}
-{{~#if bgLinear3}}
-  <g style="filter: hue-rotate({{uint16 hue}}deg);">
-    <linearGradient id="lin3" gradientTransform="rotate(90)">
-      <stop offset="0%" stop-color="rgba(8, 14, 37, 1)"></stop>
-      <stop offset="100%" stop-color="rgba(225, 114, 127, 1)"></stop>
-    </linearGradient>
-    <rect style="fill:url(#lin3)" width="2000" height="3000"></rect>
-  </g>
-{{~/if~}}
+<g style="filter: hue-rotate({{uint16 hue}}deg);">
+{{#if radial}}
+  {{#if light}}
+      <rect style="fill:hsla({{color.hue}}, {{color.saturation}}%, {{color.lightness}}%, 1)" width="2000" height="3000"></rect>
+      <radialGradient id="grad0">
+        <stop offset="0" style="stop-color:rgba(255, 255, 255, 1)"></stop>
```

Comment:
@samepant what if these were composed of skewed values from the original color inputs?

EX: instead of white here, we could have another hsla input - something like: `hue < 50 ? hue+360-50 : hue-50`, and then dial saturation and lightness up to 80-85%.

Reply:
This makes a lot of sense, will try out

Reply:
in [592d7ed](https://github.com/gnosis/wand-nft/pull/13/commits/592d7edadea14e61021b6c190438afd6f2f881ad) I switched some of the values to use the Hue of the selected color. I ended up leaving  the middle color (converted to HSL) on the radial gradient as it adds a nice consistent and similar complexity to the background across a bunch of colors.